homeassistant/components/azure_data_explorer/strings.json

```diff
@@ -2,17 +2,21 @@
   "config": {
     "step": {
       "user": {
-        "title": "Setup your Azure Data Explorer integration",
-        "description": "Enter connection details.",
         "data": {
-          "cluster_ingest_uri": "Cluster ingest URI",
-          "database": "Database name",
-          "table": "Table name",
+          "cluster_ingest_uri": "Cluster Ingest URI ",
+          "authority_id": "Authority ID",
           "client_id": "Client ID",
           "client_secret": "Client secret",
-          "authority_id": "Authority ID",
-          "use_queued_ingestion": "Use queued ingestion"
-        }
+          "database": "Database name",
+          "table": "Table name",
+          "use_queued_ingestion": "Use Queued ingestion (Mandotory for free cluster)"
+        },
+        "data_description": {
+          "cluster_ingest_uri": "Ingest-URI of the cluster, use the 'https://ingest-xxxx.xx.xx' URI",
```

Comment:
URLs should be added via the description_placeholders

Reply:
Removed the url, and just retained the hint to use -ingest url